src/apps/material/material.dev.tsx

```diff
@@ -18,7 +18,8 @@ export default {
     },
     wid: {
       name: "Work ID",
-      defaultValue: "work-of:870970-basis:52557240",
+      defaultValue: "work-of:870970-basis:06373674",
+      // defaultValue: "work-of:870970-basis:52557240",
```

Comment:
Please do not leave code in comments without context.

If this is something that we want to keep around then at least add an explanation of its relevance. An alternative to the current approach would be to define two stories: One for a "standard" material and one for a periodical and set the standard values accordingly.